modules/google_sql_database_instance/main.tf

```diff
@@ -70,12 +70,13 @@ resource "google_sql_database_instance" "instance" {
 
   settings {
 
-    tier                  = var.settings_tier
-    disk_size             = var.settings_disk_size
-    disk_autoresize       = var.settings_disk_autoresize
-    disk_type             = var.settings_disk_type
-    availability_type     = var.settings_availability_type
-    disk_autoresize_limit = var.settings_disk_autoresize_limit
+    tier                        = var.settings_tier
+    disk_size                   = var.settings_disk_size
+    disk_autoresize             = var.settings_disk_autoresize
+    disk_type                   = var.settings_disk_type
+    availability_type           = var.settings_availability_type
+    disk_autoresize_limit       = var.settings_disk_autoresize_limit
+    deletion_protection_enabled = var.deletion_protection
```

Comment:
These weren't sorted previously, but since we are touching them, let's do while here.  Can also split into smaller logical groupings based on type and sort within them to make it easier to read.